Formula/o/openapi-tui.rb

```diff
@@ -0,0 +1,36 @@
+class OpenapiTui < Formula
+  desc "TUI to list, browse and run APIs defined with openapi spec"
+  homepage "https://github.com/zaghaghi/openapi-tui"
+  url "https://github.com/zaghaghi/openapi-tui/archive/refs/tags/0.10.0.tar.gz"
+  sha256 "59ab143671843c5dc16056900b3c2413cc58a943f545ea2d94d687568410cb30"
+  license "MIT"
+  head "https://github.com/zaghaghi/openapi-tui.git", branch: "main"
+
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "openssl@3"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/openapi-tui --version")
+
+    # Fails in Linux CI with `No such device or address (os error 6)`
+    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
```

Comment:
Oh... on linux, the test cannot get the `pid` and it causes an error, my mistake :(